dev/ansible/01_generate_secrets_vault.yml

```
@@ -0,0 +1,23 @@
+- name: Generate Secrets with Vault
+  shell: |
+    export VAULT_ADDR="http://127.0.0.1:8200"
+    vault kv put secret/minio \
+    accessKey=$(vault write -field='value' gen/password length=12 symbols=0) \
+    secretKey=$(vault write -field='value' gen/password length=12 symbols=0)
+  run_once: true
```

U can use `environment` 
```suggestion
- name: Generate Secrets with Vault
  shell: |
    vault kv put secret/minio \
    accessKey=$(vault write -field='value' gen/password length=12 symbols=0) \
    secretKey=$(vault write -field='value' gen/password length=12 symbols=0)
  environment:
    VAULT_ADDR: "http://127.0.0.1:8200"
```